frontend/src/components/LandingPage/Explanation/Explanation.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import {Jumbotron} from 'reactstrap';
+import './Explanation.css';
+
+function Explanation() {
+
+    return (
+      
+    <Jumbotron className='Jumbo1' style={{"background-color": "rgb(28, 16, 38)", "background-image": ""}}>
```

Comment:
What do you think about moving the inline styling to be inside of the Explanation.css file instead of inline?

Reply:
I tried stylizing the Jumbotron in the CSS file but since it has its own default CSS properties from reactstrap those properties override the changes I want to make in the CSS file so I have to use inline styling, unfortunately.